src/chrome/content/rules/Ted.com.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="Ted">
```

Comment:
Please make additions and changes for ted.com to TED.com.xml, and more generally grep all rulesets for existing domains before making a new one.